package.json

```diff
@@ -36,17 +37,22 @@
     "body-parser": "^1.18.3",
     "cors": "^2.8.5",
     "css-loader": "^2.1.1",
+    "csv-parser": "^2.2.0",
+    "dev": "^0.1.3",
     "dotenv": "^7.0.0",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.12.1",
     "express": "^4.16.4",
     "file-loader": "^3.0.1",
+    "fs": "0.0.1-security",
```

Comment:
`fs` is built into Node.js and should not be installed separately.

Reply:
removed from `package.json`